project/Common.scala

```diff
@@ -98,4 +93,22 @@ object Common {
     // "-Yrangepos" // Removed due to slow compilation times
   )
 
+  def compilerFlags(version: String) = {
+    if (version.startsWith(scalaBinaryVersionNumber)) {
+      compilerFlagsDefault ++ Seq(
+        "-Xlint:by-name-right-associative", // By-name parameter of right associative operator.
+        "-Xlint:unsound-match", // Pattern match may not be typesafe.
+        "-Yno-adapted-args", // Do not adapt an argument list (either by inserting () or creating a tuple) to match the receiver.
+        "-Ypartial-unification", // Enable partial unification in type constructor inference
+        "-Ywarn-inaccessible", // Warn about inaccessible types in method signatures.
+        "-Ywarn-infer-any", // Warn when a type argument is inferred to be `Any`.
+        "-Ywarn-nullary-override", // Warn when non-nullary `def f()' overrides nullary `def f'.
+        "-Ywarn-nullary-unit", // Warn when nullary methods return Unit.
+        "-Ywarn-unused-import")
+    } else if (version.startsWith(scalaBinaryVersionNumber)) {
```

Comment:
This `else if` condition is the same as the one above

Reply:
Nice catch!  Fixed it ✅